src/rgw/rgw_common.h

```diff
@@ -1197,7 +1197,7 @@ struct RGWBucketInfo
   bool has_website;
   RGWBucketWebsiteConf website_conf;
 
-  RGWBucketIndexType index_type;
+  RGWBucketIndexType index_type = RGWBIType_Indexless;
 
```

Comment:
Why `RGWBIType_Indexless` ? Any justification ?

Reply:
@joscollin Can you please respond??

Reply:
`RGWBIType_Normal` is probably a better default

Reply:
@cbodley I also think `RGWBIType_Normal` is the default value, considering the above `decode()` logic.